Specs/Scene/Batched3DModel3DTileContentSpec.js

```diff
@@ -260,9 +238,9 @@ defineSuite([
         return Cesium3DTilesTester.loadTileset(scene, texturedUrl).then(function(tileset) {
             var content = tileset._root.content;
 
-            // 10 buildings, 32 ushort indices and 24 vertices per building, 8 float components (position, normal, uv) and 1 ushort component (batchId) per vertex.
-            // 10 * ((24 * (8 * 4 + 1 * 2)) + (36 * 2)) = 8880
-            var geometryByteLength = 8880;
+            // 10 buildings, 36 ushort indices and 24 vertices per building, 8 float components (position, normal, uv) and 1 uint component (batchId) per vertex.
+            // 10 * ((24 * (8 * 4 + 1 * 4)) + (36 * 2)) = 9360
+            var geometryByteLength = 9360;
```

Comment:
Due to glTF 2.0 restrictions on accessor layout, the _BATCHID accessor is now uint.

Other things that changed: textures are now 128x128 and uvs are removed from tilesets that aren't textured. Throughout there are changes to memory usage tests to account for this.